easybuild/easyconfigs/p/PyZMQ/PyZMQ-22.1.0-foss-2020b.eb

```
@@ -0,0 +1,33 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonPackage'
+
+name = 'PyZMQ'
+version = '22.1.0'
+local_zmqversion = '4.3.3'
+
+homepage = 'https://www.zeromq.org/bindings:python'
+description = """Python bindings for ZeroMQ"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

```suggestion
toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```
Use `GCCcore` instead of `foss` as the latest upstream version uses `GCCcore`. You can then also use the GCCcore version of `PyZMQ` with the `fosscuda` version of `visdom`.